CSharp/CreateMapeByWeekdayReport.cs

```diff
@@ -37,22 +37,17 @@ List<ForecastResponseJson> Forecasts
 class CreateMapeByWeekdayReport {
   // This expects to find an environment variable named MCAST_API_KEY containing the key obtained from the MCast web interface.
   // You may alternatively paste the API key on this line, though your company's policy might discourage plain-text secrets.
-  static readonly string apiKey = System.Environment.GetEnvironmentVariable("MCAST_API_KEY")!;
+  static readonly string ApiKey = System.Environment.GetEnvironmentVariable("MCAST_API_KEY")!;
```

Comment:
Standard C# practice is to use PascalCase for class fields.  We used to always do camelCase for fields that were private, and I forgot to switch to the C# standard practice on that one